src/modules/scores.js

```diff
@@ -9,4 +17,83 @@ const refreshScores = () => {
   window.location.reload();
 };
 
-refreshButton.addEventListener('click', refreshScores);
\ No newline at end of file
+// Add new score to the list and save them into localStorage
+const addScore = () => {
+  const nameInput = document.querySelector('#name').value;
+  const scoreInput = document.querySelector('#score').value;
+
+  if (nameInput && scoreInput && scoreInput !== 0 && scoreInput > 0) {
```

Comment:
- [Optional] I can see you're putting in a lot of effort to validate the user's input :rocket: :+1: . But I think there's an issue here. Name input here accepts zero as you can see in the screenshot below: 
![Screenshot from 2023-06-23 22-20-44](https://github.com/Ruthmy/leaderboardProject/assets/63087491/85264a8e-caf8-43ae-8bd9-e849da85575d)

- I don't think this creates a good user experience.
- This equally means that the nameInput field can accept an integer :smile:

Reply:
Hi @FrederickMih 
Thank you very much for your feedback!
I already made the changes to better evaluate the errors. Thanks ✨